person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    @age >= 18 || parent_permission
+  end
+
+  private :is_of_age?
```

Comment:
- [ ] **[OPTIONAL]** There is no method called `is_of_age?`. I would recommend you fix the method name in line 20.

- [ ] **[OPTIONAL]** There is no need repeating code on line 17 `(@age >= 18)`. The `of_age?` method can be used instead.

- [ ] **[OPTIONAL]** To avoid errors, on line 17 you should use the `@parent_permission` instance variable, instead of `parent_permission`.

  ```
  def of_age?
    @age >= 18
  end

  def can_use_services?
    of_age? || @parent_permission
  end
 
  private :of_age?
  ```

Reply:
@TedLivist Thanks for pointing this out, that's a silly oversight on my part; and @uchexm kindly implement all the optional suggestions for you to have error free experience in subsequent milestone. 🙏